<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>3</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>6</number>
     </property>
     <item row="9" column="1" colspan="2">
      <widget class="QTreeWidget" name="treeWidget">
       <column>
        <property name="text">
         <string>Id</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Filename</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Creation Date</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Status</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Save</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Size in Bytes</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Packages (Size/Buffersize)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Received Packages</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Buffersize</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="0" column="1" rowspan="6">
      <widget class="QPushButton" name="selectButton">
       <property name="text">
        <string>Select All</string>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="2" rowspan="6">
      <widget class="QPushButton" name="saveButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Save Selected</string>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QPushButton" name="clearAllButton">
       <property name="text">
        <string>Clear List</string>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="deselectButton">
       <property name="text">
        <string>Deselect All</string>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
